### ISSUE TITLE
Add related as another benchmarking test case?

### ISSUE THREAD

**Issue by M0r13n** [samuelcolvin/pydantic#3097]
Hi all,

I am working on a fairly large legacy code base that depends heavily on the Python library [related](https://github.com/genomoncology/related/tree/master/tests). This library is used to create nested object models that can be serialized into and de-serialized from nested Python dictionaries. It is based on the Python library `attrs`. Therefore, it can be considered as an alternative to `cattrs`.

Originally we used `related` to parse relatively small XML files (a few hundred lines). But over time these files became larger and larger, sometimes reaching 100k+ lines. Therefore, performance became an issue. So I considered switching from related to something like `pydantic` or fork `related` and improve the lib itself.

Long story short: I implemented a new benchmarking test case for `related` and think it might be useful for others. If not, feel free to close this issue/ associated PR.

With kind regards,
Leon

**Comment by PrettyWood** [comment]
Hi @M0r13n 
Thank you for your politeness (something sometimes forgotten in OSS) :)
As said on the linked PR, we can't accept this sorry.
Have a great weekend
